test/e2e/pod-scaler/local/main.go

```diff
@@ -0,0 +1,79 @@
+// +build e2e
+
+// This tool can run the pod-scaler locally, setting up dependencies in an ephemeral manner
+// and mimicking what the end-to-end tests do in order to facilitate high-fidelity local dev.
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"time"
+
+	"github.com/sirupsen/logrus"
+
+	"k8s.io/test-infra/prow/interrupts"
+
+	"github.com/openshift/ci-tools/pkg/testhelper"
+	"github.com/openshift/ci-tools/test/e2e/pod-scaler/kubernetes"
+	"github.com/openshift/ci-tools/test/e2e/pod-scaler/prometheus"
+	"github.com/openshift/ci-tools/test/e2e/pod-scaler/run"
+)
+
+func main() {
```

Comment:
Why is this a binary instead of a lib?

Reply:
Because you run `make local-pod-scaler` and have the thing running locally to e.g. interact with the API, look at the UI, whatever - it's the local dev setup

Reply:
If its a local test setup, why is it under the e2e tests?

Reply:
I just put everything in that new directory tree. Was more focused on the functionality than the placement - where should it go?

Reply:
Probably just under cmd as `fake-pod-scaler` or sth?

Reply:
I'm ambivalent and mostly just want the `make` target to work since there's dependencies that need to be around to make it functional - I'll move it over in the next round of review.